### PR TITLE
Adds filter analysis

### LINKED ISSUE #7742: Implement custom form for filter analyses
work started in branch 7742-filter-and-intersection-analysis

### PR DESCRIPTION
**Pull request by javierarce**
Closes #7742 

This PR adds the Filter analysis that allows filtering by a numeric or string column.

screen shot 2016-06-23 at 14 48 26

CR: @viddo 
CC: @xavijam

### CODE REVIEW

**Comment by viddo** [comment]
👍

File: lib/assets/locale/en.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "moran-cluster": "Moran cluster",
     "point-in-polygon": "Point in polygon",
     "filter": "Filter",
-    "kmeans": "Cluster"
+    "kmeans": "Cluster",
+    "filter-by-node-column": "Merge by column"
```
Comment (saleiva) [inline lib/assets/locale/en.json]: Wait, wrong name. This will be Link to layer

**Comment by saleiva** [comment]
The crew is so excited about this.

**Comment by javierarce** [comment]
Merging, since previous build passed and last change is just a text modification.

**Pull request merged**